discord/permissions.py

```diff
@@ -149,15 +149,16 @@ def all_channel(cls):
         """A :class:`Permissions` with all channel-specific permissions set to
         ``True`` and the guild-specific ones set to ``False``. The guild-specific
         permissions are currently:
-        - manage_emojis
-        - view_audit_log
-        - view_guild_insights
-        - manage_guild
-        - change_nickname
-        - manage_nicknames
-        - kick_members
-        - ban_members
-        - administrator
+        
+        - :attr:`manage_emojis`\n
```

Comment:
```suggestion
        * :attr:`manage_emojis`
```

Reply:
@ShineyDev Thank you for this, but why use an asterix?

Reply:
You're right. This was caused in #6476. Just remove the newlines.